Makefile

```diff
@@ -360,12 +360,34 @@ ifneq ($(origin CATALOG_BASE_IMG), undefined)
 FROM_INDEX_OPT := --from-index $(CATALOG_BASE_IMG)
 endif
 
+.PHONY: fbc-catalog-generate
+fbc-catalog-generate:
+# Step 1: Initialize operator.yaml with package info
+	$(OPM) init activemq-artemis-operator --default-channel=upstream --output yaml > fbc-catalog/operator.yaml 
```

Comment:
this generates the operator.yaml, can this be done on the fly, and there is no need to add the operator.yaml to git?
Or is the intent that this target is only executed once!